### PR TITLE
#184651078 buyer to view  their cart items

### PR DESCRIPTION
**Pull request by Kress2000**
#### What does this PR do?
Buyer should be able to view their cart
- The cart should be retrieved from the buyer's session
- The product details (name, price, image) should be retrieved from the database for each item in the cart
- The cart total should be calculated by summing the prices of the items in the cart
- The cart should be sent to the frontend to be displayed to the buyer

#### Description of Task to be completed?
- We are using a Redis server, not PostgreSQL tables, for the simplicity and making app quicker. 
- we reference the product id, and keep all product details, plus its price, and later we calculate the total price, according to the amount of quantity chosen.
- because our Redis server has been hosted in cloud, it is easy to access the server now with out highly depending on one' another.

#### How should this be manually tested?

- in swagger documentation, at the path
```
/cart
```
with the ``GET`` method, we will be able to get all items in the cart, with the  product details, simply referencing productId, and get the total price the buyer will be able to pay, by calculation of cart products wit their respective quantity

#### What are the relevant pivotal tracker/Trello stories?
#184651078

### CODE REVIEW

File: src/helpers/asyncwrapper.js
Comment (AdrineUWERA) [inline src/helpers/asyncwrapper.js:1]: Remove console logs. And we are no longer using the async wrapper. Use the express error handler.

File: src/services/twofactor.service.js
Comment (AdrineUWERA) [inline src/services/twofactor.service.js:1]: why did u change this path

File: src/tests/cartServices.test.js
Comment (AdrineUWERA) [inline src/tests/cartServices.test.js:1]: use meaningful descriptions

File: src/tests/cartItems.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      .get(`/cart`)
+      .set({ Authorization: `Bearer ${token}` });
+    expect(response).to.have.status(200);
+  });
+
+  it('should  reutrn 401, when not logged in', async function () {
+    const response = await chai.request(app).get(`/cart`);
+    expect(response).to.have.status(401);
+  });
+  it('should throw 404 error, when acesses wrong path', async function () {
```
[the remark this answers is not in the document]
Comment (Kress2000) [reply src/tests/cartItems.test.js:34]: yes, that why i got 404

File: src/services/cartActions.service.js
Comment (AdrineUWERA) [inline src/services/cartActions.service.js:1]: You should consider moving these codes to the cart.service.js file cz there are duplicates in the two files

**Review by AdrineUWERA — COMMENTED** [review]
Squash ur commits into one.

**Review by munezerobagira — APPROVED** [review]

**Review by AdrineUWERA — APPROVED** [review]

**Review by Edcreation — APPROVED** [review]

**Review by JosephNgabo — APPROVED** [review]

**Pull request merged**